src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/settings/UserDefinedMicroProfileSettings.java

```diff
@@ -142,11 +142,28 @@ public Map<String, Object> toSettingsForMicroProfileLS() {
         tools.put("validation", validation);
         validation.put("enabled", isValidationEnabled());
         validation.put("syntax", getSeverityNode(inspectionsInfo.syntaxSeverity()));
-        validation.put("unknown", getSeverityNode(inspectionsInfo.unknownSeverity()));
         validation.put("duplicate", getSeverityNode(inspectionsInfo.duplicateSeverity()));
         validation.put("value", getSeverityNode(inspectionsInfo.valueSeverity()));
         validation.put("required", getSeverityNode(inspectionsInfo.requiredSeverity()));
         validation.put("expression", getSeverityNode(inspectionsInfo.expressionSeverity()));
+
+        //Unknown properties
+        Map<String, Object> unknown = new HashMap<>();
+        validation.put("unknown", unknown);

```

Comment:
I think we could have a static methodwhich generate the severity + excluded list.

Reply:
done